src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/includeClassifier/index.test.js

```diff
@@ -106,7 +103,7 @@ describe('Include Classifier', () => {
 
     const actual = includeClassifier({
       href: 'idt3/blah',
-      pathname: ampPathname,
+      isAmpRequest: false,
```

Comment:
```suggestion
      isAmpRequest: true,
```
I think this should be true?